### PR TITLE
#161044635 View travel checklist for 'Open' requests

### PR DESCRIPTION
**Pull request by rovilay**
#### What does this PR do?
This PR displays the travel checklists for open requests

#### Description of Task to be completed?
- Add the `travel checklist` button to `request table menu`
- Display travel checklist on a modal when `travel checklist button` is clicked

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Run the server `make start`
- Login and make a `new request`
- Click on the `new request` ellipsis and then click on `travel checklist` on the modal dropdown
- The travel checklist modal should pop up and show the checklist for the travel request.

#### Any background context you want to provide?
- You need to clone the [backend](https://github.com/andela/travel_tool_back.git) to test this functionality.

#### What are the relevant pivotal tracker stories?
[#161044635](https://www.pivotaltracker.com/story/show/161044635)

#### Screenshots (if appropriate)
screen shot 2018-10-13 at 6 22 22 pm

screen shot 2018-10-13 at 6 23 23 pm

#### Questions:
N/A

### CODE REVIEW

File: src/components/Table/index.jsx [hunk 1]
```diff
@@ -207,6 +209,28 @@ export class Table extends Component {
     );
   }
 
+  renderTravelCheckListModal() {
+    const { closeModal, shouldOpen, modalType, travelChecklists } = this.props;
+    const { menuOpen: { id } } = this.state;
+    return (
+      <Modal
+        closeModal={closeModal}
+        width="600px"
```
Comment (ezrqnkemboi) [inline src/components/Table/index.jsx]: Using the px parameter in width is good and popular. The one limitation I see is based on responsiveness especially when screen sizes changes. I would have suggested if you could have used % instead. That is, 600px/1440px, which gives approximately 41.7% as the width. Think about it and share your thoughts with me.

Comment (rovilay) [reply src/components/Table/index.jsx]: @ezrqnkemboi  Thanks for the feedback. But the `width props` is a value for `max-width` parameter. So the modal fits based on the viewport and is responsive by default.

**Comment by seunzone** [comment]
@rovilay kindly fix all merge conflicts

File: src/components/Table/index.jsx (hunk 1 above)
Comment (rovilay) [reply src/components/Table/index.jsx]: @ezrqnkemboi  please mark this conversation as resolved.

**Pull request closed without merging**